### DISCUSSION TITLE
Render without clearing screen first

### DISCUSSION THREAD

**Discussion by esperanc** [cables-gl/cables_docs#620] [Answers & Questions] [answered]
I am trying to create a really simple project where a circle is animated randomly, but the screen is always cleared at each frame. I tried unchecking the "Clear" and "ClearAlpha" buttons in mainloop, to no avail.

https://cables.gl/p/egSqsM

**Comment by marioepsley** [comment]
one way https://cables.gl/edit/HgXBuL

**Comment by pandrr** [comment, marked as answer]
currently only possible with rendering to texture, the canvas is not created using "preservedrawingbuffer" and there is (currently) no way to change that afterwards. 
using a texture gives you more control anyway and is a good way to do it.

**Comment by esperanc** [comment]
Thanks for the feedback. Cheers!

--Claudio
